init.php

```diff
@@ -344,6 +344,11 @@ public function hook_prefs_save_feed($feed_id)
 		$this->host->set($this, "options_feeds", $options_feeds);
 	}
 
+	public function db_escape_string($s, $strip_tags = true) {
+		if ($strip_tags) $s = strip_tags($s);
+		return pg_escape_string($s);
```

Comment:
pgsql module may be not loaded

Reply:
Fair enough, I will bring Mysql logic back